src/openms/include/OpenMS/CHEMISTRY/ProteaseDigestion.h

```diff
@@ -72,11 +72,11 @@ namespace OpenMS
     /// Returns the number of peptides a digestion of @p protein would yield under the current enzyme and missed cleavage settings.
     Size peptideCount(const AASequence& protein);
 
-    /// Variant of EnzymaticDigestion::isValidProduct() with support for methionine cleavage
-    bool isValidProduct(const String& protein, Size pep_pos, Size pep_length, bool ignore_missed_cleavages = true, bool methionine_cleavage = false) const;
+    /// Variant of EnzymaticDigestion::isValidProduct() with support for n-term protein cleavage and random D|P cleavage
+    bool isValidProduct(const String& protein, Size pep_pos, Size pep_length, bool ignore_missed_cleavages = true, bool allow_nterm_protein_cleavage = false, bool allow_random_asp_pro_cleavage = true) const;
```

Comment:
one last question.... is allow_random_asp_pro_cleavage = true a sensible default?

Reply:
I dont think so, better to make it false by default